### PR TITLE
Adding a security page

### PR DESCRIPTION
**Pull request by flossypurse**
Adding general Security page per our compliance needs.

### CODE REVIEW

File: src/pages/security.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+              <p>Temporal Technologies Inc was issued a clean SOC2 Type 1 report on Janurary 31st 2021 from <a href="https://www.connor-consulting.com/">Connor Consulting</a></p>
+              <div className={styles.justifyCenter}>
+                <img className={styles.soc2Logo} src={useBaseUrl("img/21972-312_SOC_NonCPA_Blk.png")} alt="SOC2 Logo" />
+              </div>
+              <p>To request a copy of the report, contact us.</p>
+            </div>
+          </div>
+          <div className="row">
+            <div className={clsx('col col--6', styles.justifyRight)}>
+              <h2>Disclosure</h2>
```
Comment (mcbryde) [inline src/pages/security.js:58]: Update to "Responsible Disclosure"?

File: src/pages/security.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+            </div>
+            <div className={clsx('col col--4', styles.justifyLeft)}>
+            <p>If you have any concerns about security or would like to report a security issue, please reach out to our team at <a href="mailto:security@temporal.io">security@temporal.io</a>.</p>
+            <p>Include the details of your concerns or finding and we will get back to you.</p>
```
Comment (mcbryde) [inline src/pages/security.js:62]: Replace with: 

We promise not to bring legal action against people who:
- Share with us the full details of any problem they've found
- Keep the issue private until we've had a reasonable time to address it
- Don't intentionally harm our service or exfiltrate data from it

**Review by mcbryde — COMMENTED** [review]
A few text suggestions, to add text on responsible disclosure.

File: src/pages/security.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            </div>
+          </div>
+          <div className="row">
+            <div className={clsx('col col--6', styles.justifyRight)}>
+              <h2>Encryption</h2>
+            </div>
+            <div className={clsx('col col--4', styles.justifyLeft)}>
+              <p>All traffic to and from the Temporal Cloud is encrypted with mTLS.</p>
+              <p>Customers have access to data encryption tools to encrypt data at rest.</p>
+            </div>
```
Comment (sw-yx) [inline src/pages/security.js:42]: perhaps add a link to further details here https://docs.temporal.io/docs/server/security

i am 10% worried that having two "Security pages" will be confusing if we don't handle the communication right. this one is for compliance folks, the other is for developers looking to lock down temporal.

**Review by mcbryde — APPROVED** [review]
LGTM

**Pull request merged**